.github/scripts/feature_flag_monthly_audit.py

```
@@ -0,0 +1,307 @@
+        if entry["removal_comment"] else ""
+    )
+ 
+    # Jira description in Atlassian Document Format (ADF)
+    description_adf = {
+        "type": "doc",
+        "version": 1,
+        "content": [
+            {
+                "type": "paragraph",
+                "content": [
+                    {"type": "text", "text": f"The feature flag "},
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)